src/rate_limiter.cc

```diff
@@ -78,8 +78,8 @@ RateLimiter::RegisterModelInstance(
       // Without this serialization instances of other models might fail
       // to load because of the resource constraints in this instance.
       std::lock_guard<std::mutex> lk(resource_manager_mtx_);
-      resource_manager_->AddModelInstance(pair_it.first->second.get());
-      const auto& status = resource_manager_->UpdateResourceLimits();
+      const auto& status =
+          resource_manager_->AddModelInstance(pair_it.first->second.get());
       if (!status.IsOk()) {
         resource_manager_->RemoveModelInstance(pair_it.first->second.get());
```

Comment:
@kthui it looks like we throw away the status returned by `RemoveModelInstance`, which in turn calls several other methods that can return detailed status errors. 

Can we propogate the error statuses back up via `LOG_ERROR` in the places where we call this method? It looks like we call it in a few places.

Reply:
Yes. Updated: [Minor fix and return error on instance remove failure](https://github.com/triton-inference-server/core/pull/208/commits/33a5369309a117e216614f91ca96df7b95cadedb)